cmd/csi-sanity/sanity_test.go

```diff
@@ -50,5 +50,5 @@ func TestSanity(t *testing.T) {
 	if len(endpoint) == 0 {
 		t.Fatalf("--%sendpoint must be provided with an CSI endpoint", prefix)
 	}
-	sanity.Test(t, endpoint, mountPoint)
+	sanity.Test(t, endpoint, mountPoint, "")
```

Comment:
Maybe use a mountdir arg. @sbezverk said to check "islikelynotmountpoint" function to see if that could cause issues. @lpabon also suggests changing this into a test config.